assemblies/assembly-getting-started-business-optimizer.adoc

```diff
@@ -44,30 +44,7 @@ include::{enterprise-dir}/optimizer/optimizer-running-the-examples-in-an-IDE-pro
 // include::{enterprise-dir}/optimizer/examples-table-ref.adoc[leveloffset=+2]
 
 
-include::{enterprise-dir}/employee-rostering/workbench-er-tutorial-con.adoc[leveloffset=+1]
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-example-deploying-in-central-proc.adoc[leveloffset=+2]
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-example-recreate-con.adoc[leveloffset=+2]
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-example-setup-proc.adoc[leveloffset=+3]
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-facts-vs-entities-con.adoc[leveloffset=+3]
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-data-model-proc.adoc[leveloffset=+3]
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-planning-entity-proc.adoc[leveloffset=+4]
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-planning-solution-proc.adoc[leveloffset=+4]
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-constraints-con.adoc[leveloffset=+3]
-include::{drools-dir}/AuthoringAssets/drl-rules-con.adoc[leveloffset=+4]
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-optimizer-drools-rules-proc.adoc[leveloffset=+4]
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-guided-rule-editor-proc.adoc[leveloffset=+3]
-include::{drools-dir}/AuthoringAssets/guided-rules-con.adoc[leveloffset=+4]
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-balance-employees-shift-rule-proc.adoc[leveloffset=+4]
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-one-shift-per-day-rule-proc.adoc[leveloffset=+4]
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-shift-required-skills-met-proc.adoc[leveloffset=+4]
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-day-off-request-rule-proc.adoc[leveloffset=+4]
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-solver-configuration-proc.adoc[leveloffset=+3]
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-solver-termination-configuration-proc.adoc[leveloffset=+3]
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-using-rest-proc.adoc[leveloffset=+2]
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-register-solver-proc.adoc[leveloffset=+3]
-include::{enterprise-dir}/employee-rostering/wb-employee-rostering-call-solver-proc.adoc[leveloffset=+3]
-
-
+include::assembly-optaplanner-employee-rostering.adoc[]
```

Comment:
the same as https://github.com/emmurphy1/kie-docs/pull/352/commits/d4aa0377bdd5a18aa43671af3ca15bfb9a835abe#r833128758

Reply:
@dupliaka this is just for the PAM/DM Getting Started titles. The idea is to give users a walk-through tutorial of the different aspects of the product. Previously, the entire Getting Started section from the OP documentation was duplicated here. Now, I am just including the Employee Rostering tutorial in Business Central.

Reply:
ok